clash-prelude/src/Clash/Signal.hs

```diff
@@ -97,7 +97,6 @@ module Clash.Signal
   , ResetKind(..)
   , SResetKind(..)
   , ResetPolarity(..)
-  , SResetPolarity(..)
```

Comment:
I completely missed this. I think it was a mistake to remove this export? If not, it should be documented.

Reply:
Yup, thanks!